packages/api-clients/open-api/delegationApi.yml

```diff
@@ -830,6 +901,23 @@ components:
         - status
         - title
         - errors
+    Pagination:
+      type: object
+      additionalProperties: false
+      properties:
+        offset:
+          type: integer
+          format: int32
+        limit:
+          type: integer
+          format: int32
+        totalCount:
+          type: integer
+          format: int32
+      required:
+        - offset
+        - limit
+        - totalCount
```

Comment:
> Usually in other process services when we return paginated data we return only `{ results: T, totalCount: number }` without also returning offset and limit (examples [here](https://vscode.dev/github/pagopa/interop-be-monorepo/blob/agreement-activation-contract-fixes/packages/api-clients/open-api/catalogApi.yml#L1733-L1734) and [here](https://vscode.dev/github/pagopa/interop-be-monorepo/blob/agreement-activation-contract-fixes/packages/api-clients/open-api/agreementApi.yml#L1221-L1222))

True but in this case the [SRS](https://pagopa.atlassian.net/wiki/spaces/PDNDI/pages/1236959344/SRS+Incaricato#Delegato-crea-finalit%C3%A0) required `limit` and `offset` too, we could propose to remove them if not needed

Reply:
I think it was a typo/leftover in the SRS, because that's the format we use in the BFF but never in the *-process services - if you agree I would keep it as it is in the BFF but adapt it to how we do in other process services here